src/workerd/io/worker.c++

```diff
@@ -1194,6 +1194,17 @@ Worker::Isolate::Isolate(kj::Own<Api> apiParam,
                   v8::Local<v8::Value> argument)> callback) -> v8::Maybe<void> {
         try {
           auto& js = jsg::Lock::from(isolate);
+
+          // TODO(soon): Hopefully this logging is temporary. We want to get an idea
+          // of where the cross request promises are being resolved just in general.
+          // To do so we need to try to capture a stack. We do so by creating an error
+          // object and logging it.
+          auto err = v8::Exception::Error(js.str(""_kj)).As<v8::Object>();
```

Comment:
Should we wrap this in a block with a HandleScope?

Reply:
we can, sure.